### PR TITLE
Support for custom config path option

### PR DESCRIPTION
**Pull request by milindgoel15**
Closes https://github.com/OutdatedGuy/package_rename/issues/37

- Added path option to provide a custom location for package_rename_config.yaml file
- Updated readme
- Added basic help command that prints out command usages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,71 +21,85 @@ For more info see [list of changed fields](CHANGED_FIELDS.md)
...
+
+   web:
+      app_name: # (String) The title and display name of the web app and PWA
+      description: # (String) The description of the web app and PWA
+
+   windows:
+      app_name: # (String) The window title & software name of the windows app
+      organization: # (String) The organization name (company name) of the windows app
+      copyright_notice: # (String) The legal copyright of the windows app
+      exe_name: # (String) The executable name (binary name) of the windows app
```
Comment (OutdatedGuy) [inline README.md:69]: Remove all the extra added spacings

File: README.md [hunk 2]
```diff
@@ -21,71 +21,85 @@ For more info see [list of changed fields](CHANGED_FIELDS.md)
...
+```
+
 ## Flavour Support
 
 Package Rename supports flavours. You can add flavour specific configurations by adding `flavour_name` in configuration key.
 
 ```yaml
 package_rename_config-flavour_name:
-  # ...
+   # ...
```
Comment (OutdatedGuy) [inline README.md:102]: Remove extra spacing

File: README.md [hunk 3]
```diff
@@ -94,6 +108,17 @@ And then run the following command:
 
 ```bash
 dart run package_rename --flavour=flavour_name
+
+or
+
+dart run package_rename -f flavour_name
+
+```
```
Comment (OutdatedGuy) [inline README.md:116]: Add this in separate code block so people can copy it easily

**Review by OutdatedGuy — CHANGES_REQUESTED** [review]
@milindgoel15, thanks for the PR. I just need a few mentioned changes here.

File: README.md [hunk 4]
```diff
@@ -21,71 +21,85 @@ For more info see [list of changed fields](CHANGED_FIELDS.md)
 
 ```yaml
 dev_dependencies:
-  package_rename: ^1.4.0
+   package_rename: ^1.4.0
```
[the remark this answers is not in the document]
Comment (milindgoel15) [reply README.md:24]: Yea all the spaces happened due to my vscode indent settings. You probably use 2 space sizes, while I use 3.

I don't think it causes any issue but still I will revert the spaces.

**Review by OutdatedGuy — APPROVED** [review]
LGTM!

**Pull request merged**